_posts/2025/2025-02-05-ordinal-numbers-in-rails-1st-2nd-3rd-4th.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Ordinal Numbers in Rails: 1st, 2nd, 3rd, 4th"
+description: "1st, 2nd, 3rd, 4th: turning numbers into their 'spoken' equivalents with ActiveSupport"
+layout: article
+category: ruby
+date: 2025-02-05 02:00
+image:
+  base: "2025/ordinal-numbers-in-rails"
+  alt: "Black & white trophy"
+  credit: "Giorgio Trovato (edited)"
+  source: "https://unsplash.com/photos/yellow-and-white-trophy-_XTY6lD8jgM"
+---
+
+When developing Rails applications, you often need to present numbers in a more human-readable format. For instance, you might want to display "1st" instead of just "1" or "42nd" rather than "42". Rails provides a neat solution for this through ActiveSupport's Core extensions.
```

Comment:
```suggestion
When developing Rails applications, you often need to present numbers in a more human-readable format. For instance, you might want to display "1st" and "42nd" instead of just "1" or "42". Rails provides a neat solution for this through Active Support's Core extensions.
```